src/controller/AdminController.php

```diff
@@ -154,7 +155,7 @@ public function displayRefusedComments(UserModel $userModel, PostModel $postMode
 		$refuseComments = $commentModel->loadRefuseComments();
 
 		// display 
-		$this->render('admin_refused_comments.twig', $_SESSION, [], [], [], $refuseComments);
+		$this->render('admin_refused_comments.twig', [], [], [], [], $refuseComments, $_SESSION);
```

Comment:
```suggestion
		$this->render('admin_refused_comments.twig', $_SESSION, [], [], [], $refuseComments);
```

Reply:
Modif ok : 
$this->render('admin_refused_comments.twig', $_SESSION, [], [], [], [], $refuseComments);